tests/x11regressions/evince/evince_find.pm

```diff
@@ -0,0 +1,37 @@
+use base "x11test";
+use strict;
+use testapi;
+
+# Case 1436022 - Evince: Find
+sub run() {
+    my $self = shift;
+    x11_start_program("evince " . autoinst_url . "/data/x11regressions/test.pdf");
+
+    send_key "ctrl-f";    # show search toolbar
+    assert_screen 'evince-search-toolbar', 5;
+
+    my $keyword = 'To search for';
+    type_string $keyword;
+    sleep 5;    # searching automatically
+    assert_screen 'evince-search-1stresult', 5;
+
+    for ( 1..2 ) {
+        send_key "ctrl-g";    # go to next result
+        sleep 1;
+    }
+    assert_screen 'evince-search-3rdresult', 5;
+
+    for ( 1..2 ) {
+        send_key "ctrl-shift-g";    # go to previous result
+        sleep 1;
+    }
+    assert_screen 'evince-search-1stresult', 5;
+    
+    send_key "esc";
+    sleep 1;
+    send_key "ctrl-w";
+    sleep 1;
```

Comment:
this is unnecessary, x11test will check assert_screen('generic-desktop') in the end ie. the post hook.